packages/mobile/src/pincode/PincodeEnter.tsx

```diff
@@ -115,16 +116,16 @@ class PincodeConfirmation extends React.Component<Props, State> {
   }
 
   onPressConfirm = () => {
+    const { fornoMode, navigation, currentAccount } = this.props
     const { pin } = this.state
-    const fornoMode = this.props
-    if (this.props.currentAccount && !fornoMode) {
+    const withVerification = navigation.getParam('withVerification')
+    // TODO: Implement PIN verification for forno mode
```

Comment:
Can we do this as part of this PR? Otherwise, is there a task open already for it?

Reply:
I've implemented it as a part of this PR